weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableColumns.tsx

```diff
@@ -327,6 +331,56 @@ function buildCallsTableColumns(
   );
   cols.push(...newCols);
 
+  // Create special feedback columns with grouping model
+  const feedbackCols = allDynamicColumnNames.filter(c =>
+    c.startsWith('feedback.')
+  );
+  if (feedbackCols.length > 0) {
+    const convertFeedbackFieldToBackendFilter = (field: string) => {
+      // feedback.wandb.annotation.Text-field-2.value.Text-field-2.ZQUEOy2FtgkRWahm8ucKgGHQGQYhoroXSug6SY6SSZQ
+      // -> feedback.[wandb.annotation.Text-field-2].value.Text-field-2.ZQUEOy2FtgkRWahm8ucKgGHQGQYhoroXSug6SY6SSZQ
+      const regex = /feedback\.wandb\.([^.]+\.[^.]+)\.value\./;
+      const feedbackType = regex.exec(field)?.[1] ?? field; // annotation.my-type
+      const afterType = field.split(feedbackType).pop();
+      return `feedback.[wandb.${feedbackType}]${afterType}`;
+    };
+
+    // Add feedback group to grouping model
+    groupingModel.push({
+      groupId: 'feedback',
+      headerName: 'feedback',
+      children: feedbackCols.map(col => ({
+        field: convertFeedbackFieldToBackendFilter(col),
+      })),
+    });
+
+    const getFeedbackHeaderFromField = (field: string) => {
+      // feedback.wandb.annotation.Text-field-2.value.Text-field-2.ZQUEOy2FtgkRWahm8ucKgGHQGQYhoroXSug6SY6SSZQ
+      // -> Text-field-2
+      const regex = /feedback\.wandb\.[^.]+\.([^.]+)\.value\./;
+      const match = regex.exec(field);
+      return match?.[1] ?? field;
+    };
+
+    // Add feedback columns
+    const feedbackColumns: Array<GridColDef<TraceCallSchema>> =
+      feedbackCols.map(c => ({
+        field: convertFeedbackFieldToBackendFilter(c),
+        headerName: `Feedback.${getFeedbackHeaderFromField(c)}`,
```

Comment:
Making this comment here. Consider it a nit: It might be nice to have the group header for annotation say "Annotations"?